### PR TITLE
Add contract language to documentation page

### LINKED ISSUE #1025: Publish sample contract language for 21st Century IDEA
This issue is a follow-up to #995. 

**Remaining to-do list includes:**
- [X] Content located [here](https://docs.google.com/document/d/1Dd5q9wgzSIqdgNcvslMPVFXPdYGEVLIX0HRvjuRy_JM/edit?usp=sharing)
- [X] Decide where to publish it. Publish on designsystem.digital.gov under `How to Use`
- [x] Create Github page
- [x] KPI: engagement w content. Metrics:  time on page - sessions - entrance - exit - 
- [x] Publish on designsystem.digital.gov as a new documentation page under `How to Use`

### PR DESCRIPTION
**Pull request by allie-shaw**
## Description
Draft of sample contract page based on [Sample Contract Language for the 21st Centure IDEA)(https://docs.google.com/document/d/1Dd5q9wgzSIqdgNcvslMPVFXPdYGEVLIX0HRvjuRy_JM/edit?ts=5f43abe4#)

[Preview Link](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/as-add-contract-lang/documentation/sample-contract-language/)

Closes #1025 

## Additional information

Include any of the following (as necessary): 

* Relevant research and support documents
* Type of content review needed: stylistic or copy editing
* Screenshot images
* Notes
* etc.

Before you hit Submit, make sure you’ve done whichever of these applies to you:

- [ ] Follow the [18F Front End Coding Style Guide](https://pages.18f.gov/frontend/) and [Accessibility Guide](https://pages.18f.gov/accessibility/checklist/).
- [ ] Run `npm test` and make sure the tests for the files you have changed have passed.
- [ ] Run your code through [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/) and make sure it’s error free.
- [ ] Title your pull request using this format: [Website] - [UI component]: Brief statement describing what this pull request solves.

### CODE REVIEW

File: _data/nav.yml [hunk 1]
```diff
@@ -61,6 +61,7 @@ How to use USWDS:
   - href: /documentation/video-tutorials/
   - href: /documentation/implementations/
   - href: /documentation/showcase/
+  - href: /documentation/sample-contract/
```
Comment (mejiaj) [inline _data/nav.yml:64]: @allie-shaw, I've added the page to the sidenav in `How to use USWDS`

image

**Comment by mejiaj** [comment]
@thisisdano this is ready for review, but depends on the Security page #1032 to get merged.

File: pages/documentation/getting-started/sample-contract-language.md [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+
+Then work with your contracting officer (CO) to determine your acquisition strategy, including customizing the draft language below to meet your needs and requirements.
+
+## Draft contract language
+
+### New solicitations
+
+In new solicitations, you can include requirements for 21st Century IDEA in the performance work statement.
+
+#### Draft language for new solicitations
```
Comment (thisisdano) [inline pages/documentation/getting-started/sample-contract-language.md:39]: @afeijoo I've been formatting this page and I have a few instances where I'm not sure if the heading is part of the contract language or meant to introduce it. Can you look at each of these (this is the first one) and let me know if the heading should be in the blockquote or outside of it.

File: pages/documentation/getting-started/sample-contract-language.md [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+{:.example.border-left-05.border-base-light.padding-left-105.margin-top-2.measure-5}
+> Any website that is developed, modernized, enhanced, maintained, or otherwise delivered as a result of this contract shall be in compliance with the website standards of the Technology Transformation Services of the General Services Administration at [https://designsystem.digital.gov/website-standards/](https://designsystem.digital.gov/website-standards/).
+>
+> The U.S. Web Design System (USWDS) shall be adopted incrementally over the life of the requirement and the contractor shall prioritize implementation to align with the priorities identified within the Performance Work Statement.
+
+### Open solicitations
+
+You can ask your CO to amend open solicitations to include requirements for 21st Century IDEA.
+
+#### Draft language for open solicitations
```
Comment (thisisdano) [inline pages/documentation/getting-started/sample-contract-language.md:50]: @afeijoo This one too.

File: pages/documentation/getting-started/sample-contract-language.md [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+> The U.S. Web Design System (USWDS) shall be adopted incrementally over the life of the requirement and the contractor shall prioritize implementation to align with the priorities identified within the Performance Work Statement.
+
+### Customize the draft language
+
+Work with your CO to customize the draft language for new and open solicitations. For example, instead of “life of the requirement” you may use period of performance, sprint, next refresh, or another time period. Or, you may replace “Performance Work Statement” with “Statement of Objectives,” agency modernization plans, or another source of priorities.
+
+## Evaluation factors
+You can use evaluation factors to select the best vendor by evaluating their ability to develop modern websites that meet specific requirements in 21st Century IDEA, including incremental adoption of USWDS.
+
+### Draft 1: <span class="text-light">Modern website or digital service evaluation factor</span>
```
Comment (thisisdano) [inline pages/documentation/getting-started/sample-contract-language.md:64]: @afeijoo And this one.

File: pages/documentation/getting-started/sample-contract-language.md [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+   </tr>
+   <tr>
+    <th scope="row">Mobile-Friendly</th>
+    <td>be functional and usable on mobile devices</td>
+   </tr>
+  </tbody>
+ </table>
+</div>
+
+### Draft 2 (scenario): <span class="text-light">Modern website or digital service scenario evaluation factor</span>
```
Comment (thisisdano) [inline pages/documentation/getting-started/sample-contract-language.md:117]: @afeijoo And finally this one. Thank you!

File: pages/documentation/getting-started/sample-contract-language.md (hunk 2 above)
Comment (thisisdano) [reply pages/documentation/getting-started/sample-contract-language.md:39]: [Preview link for reference](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/as-add-contract-lang/documentation/sample-contract/) →

**Pull request merged**